plugins/microsoft_log_analytics/help.md

```diff
@@ -271,13 +520,21 @@ Example output:
 |Name|string|False|Table's name|
 |Rows|[]object|False|Table's rows|
 
+#### tag
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Name|string|False|Tag's name|
+|Value|string|False|Tag's value|
+
 
 ## Troubleshooting
 
 _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.2.0 - Added new actions Create or Update Saved Search, Get Saved Search, Delete Saved Search, and List All Saved Searches
 * 1.1.0 - Search trigger thats run Log Analytics query every interval time
```

Comment:
```suggestion
* 1.2.0 - Add new actions Create or Update Saved Search, Get Saved Search, Delete Saved Search, and List All Saved Searches
* 1.1.0 - Search trigger that run Log Analytics query every interval time
```

Reply:
Updated the change for changelog 1.2.0. Not changing the description for 1.1.0 as it is already released.